src/components/Navbar/Notifica/styles.scss

```diff
@@ -0,0 +1,38 @@
+@import '~base-scss/base';
+
+.notifica {
+    position: relative;
+
+    .count {
+        @include background-color($color-danger);
+        display: flex;
+        position: absolute;
+        top: $spacing-small;
+        right: $spacing-small;
+        align-items: center;
+        justify-content: center;
+        border-radius: ($font-size-extra-small + $spacing-extra-small * 2) / 2;
+        padding: $spacing-extra-small;
+        min-width: $font-size-extra-small + $spacing-extra-small * 2;
+        font-size: $font-size-extra-small;
+        font-weight: $font-weight-bold;
+    }
+
+    .dropdown-menu {
+        width: 100%;
+        height: 100%;
+    }
+}
+
+.notification-dropdown {
+    min-width: 420px;
```

Comment:
variables?

Reply:
is for rookies

Reply:
You are a rookie as well, no go set your varibables

Reply:
*now
*variables